packages/application-shell/src/components/app-bar/app-bar.js

```diff
@@ -12,79 +12,93 @@ import { REQUESTS_IN_FLIGHT_LOADER_DOM_ID } from '../requests-in-flight-loader/c
 import messages from './messages';
 import styles from './app-bar.mod.css';
 
-const AppBar = props => (
-  <div className={styles['app-bar']} data-test="top-navigation">
-    <Spacings.Inline>
-      <div className={styles.logo}>
-        <img src={LogoSVG} className={styles['logo-img']} alt="Logo" />
-      </div>
+const AppBar = props => {
+  const previouslyUsedProjectKey = selectProjectKeyFromLocalStorage();
+  return (
+    <div className={styles['app-bar']} data-test="top-navigation">
+      <Spacings.Inline>
+        <div className={styles.logo}>
+          {!props.user ? (
+            <LoadingPlaceholder shape="square" size="l" />
```

Comment:
Can't we always render the logo? I find it weird to have a placeholder for the logo

Reply:
Yeah I also find this weird

Reply:
Yeah but did you guys read my comment here https://github.com/commercetools/merchant-center-application-kit/pull/183#discussion_r239092501 ?

We are using user information which can be loading, that's why this rendering case 🤔

Reply:
The "loading" is just for the link, not for the logo. Just render the image twice and we're fine. No loading spinner

Reply:
So true 🤦‍♂️

Reply:
Any objections on keeping the `square` shape @emmenko or should I remove it?